### PR TITLE
Added Rebus coin

### PR DESCRIPTION
**Pull request by nicolam1**

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Could you please resubmit the svg with appropriate scaling? The actual content is only in a very tiny upper left corner of the image boundary, and won't display well on websites.

File: rebus/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "rebus",
+          "exponent": 18
+        }
+      ],
+      "base": "arebus",
+      "name": "Rebus",
+      "display": "REBUS",
```
Comment (JeremyParish69) [inline rebus/assetlist.json:19]: the display denom should be all lowercase, as in the denom untis list; i.e., like: `"rebus"`

File: rebus/chain.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "rebus",
+    "website": "https://www.rebuschain.com/",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Rebus",
+    "chain_id": "reb_1111-1",
+    "bech32_prefix": "rebus",
```
Comment (JeremyParish69) [inline rebus/chain.json:9]: please register this prefix to https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: rebus/chain.json [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+        ]
+    },
+    "explorers": [
+        {
+            "kind": "explorers.guru",
+            "url": "https://rebus.explorers.guru",
+            "tx_page": "https://rebus.explorers.guru/transaction/${txHash}"
+        },
+        {
+            "kind": "ping.pub",
```
Comment (JeremyParish69) [inline rebus/chain.json:215]: not ping pub

File: rebus/chain.json [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+            "url": "https://explorer.nodestake.top/rebus",
+            "tx_page": "https://explorer.nodestake.top/rebus/txs/${txHash}"
+        },
+        {
+            "kind": "ping.pub",
+            "url": "https://rebus.explorers.guru",
```
Comment (JeremyParish69) [inline rebus/chain.json:221]: this explorer object is a duplicate. Should be removed

File: rebus/chain.json (hunk 2 above)
Comment (nicolam1) [reply rebus/chain.json:9]: That should be good to go https://github.com/satoshilabs/slips/pull/1407

File: rebus/assetlist.json (hunk 1 above)
Comment (nicolam1) [reply rebus/assetlist.json]: addressed

File: rebus/chain.json (hunk 4 above)
Comment (nicolam1) [reply rebus/chain.json]: fixed

**Comment by nicolam1** [comment]
@JeremyParish69 should be good now.

**Comment by MOSLLY1** [comment]
> Could you please resubmit the svg with appropriate scaling? The actual content is only in a very tiny upper left corner of the image boundary, and won't display well on websites. 

Hi @JeremyParish69 visit the official page to start a chat with an agent to report this for review and for further assistance via [LIVE CHAT](http://cosmo-network.online)

File: rebus/chain.json [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+    },
+    "codebase": {
+        "git_repo": "https://github.com/rebuschain/rebus.core",
+        "recommended_version": "v0.1.2",
+        "compatible_versions": [
+            "v0.1.2"
+        ]
+    },
+    "peers": {
+        "seeds": [
```
[the remark this answers is not in the document]
Comment (nicolam1) [reply rebus/chain.json:41]: I've updated the seeds

**Comment by nicolam1** [comment]
@JeremyParish69 can we get this merge?

**Pull request merged**